src/components/Community/GrowthSection/CommunityNavItem.tsx

```diff
@@ -0,0 +1,99 @@
+import { styled } from "styled-components";
+
+import { IcArrowrightRg } from "../../../assets";
+import { communityNav } from "../../../constatns/Community/communityConstants";
+
+function CommunityNavItem() {
+  return (
+    <CommunityNavItemWrapper>
+      {communityNav.map((item) => (
+        <CommunityNavContainer key={item.label} type="button" backgroundColor={item.backgroundColor}>
+          <LabelBox>
+            {item.label}
+            <IcArrowrightRgIcon />
+          </LabelBox>
+          <ContentContainer>
+            <Content>{item.content1}</Content>
+            <Content>{item.content2}</Content>
+          </ContentContainer>
+          <ImgContainer>{item.element}</ImgContainer>
+        </CommunityNavContainer>
+      ))}
+    </CommunityNavItemWrapper>
+  );
+}
+
+export default CommunityNavItem;
+
+const CommunityNavItemWrapper = styled.section`
+  display: flex;
+
+  gap: 2.716rem;
+`;
+
+const CommunityNavContainer = styled.button<{ backgroundColor: string }>`
+  display: flex;
+  flex-direction: column;
+
+  width: 19.1rem;
+  height: 20rem;
+
+  border-radius: 0.8rem;
+
+  background-color: ${({ theme, backgroundColor }) =>
+    backgroundColor === "yellow"
+      ? theme.colors.o01
+      : backgroundColor === "orange"
+        ? theme.colors.o03
+        : backgroundColor === "skyBlue"
+          ? theme.colors.bb01
+          : theme.colors.bb02};
```

Comment:
이렇게 조건절이 많은 경우는 가독성을 위해 switch문도 고려해볼 수 있습니다!!
```suggestion
background-color: ${({ theme, backgroundColor }) => {
    switch (backgroundColor) {
      case "yellow":
        return theme.colors.o01;
      case "orange":
        return theme.colors.o03;
      case "skyBlue":
        return theme.colors.bb01;
      default:
        return theme.colors.bb02;
    }
```

Reply:
헉 확실히 더 가독성이 좋네요...!! 맞춰서 수정하겠습니다!